packages/frontend/src/hooks/useApprove.ts

```diff
@@ -61,12 +61,13 @@ const useApprove = token => {
           network: {
             slug: token.chain?.slug,
             networkId: token.chain?.chainId,
-          }
-        }
+          },
+        },
       },
       onConfirm: async (approveAll: boolean) => {
+        const overrides = await token.txOverrides(chain)
         const approveAmount = approveAll ? constants.MaxUint256 : amount
-        return token.approve(spender, approveAmount)
+        return token.approve(spender, approveAmount, overrides)
```

Comment:
The overrides parameter here in token approve shouldn't be necessary since the sdk token approve function calls populateApproveTx which already includes the overrides. I think the overrides parameter can be removed because of that reason